### ISSUE TITLE
Add number of total reviews and total recent reviews

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1455]
I love buying cheap bundles but usually some of the bundles have really unknown games so a good indicator sometimes of a hidden gem is the amount of reviews and if it is positive or not.

You guys already have a great way to see the % of reviews but no way to know if 5 people reviewed the game or 500.

So I think an additional column maybe to show total reviews and maybe total recent reviews might be a cool idea to help gauge big bundles with tons of smaller games.

Basically more meta data when showing bundles and lists of games to give us more ability to discern the quality of a game at a glance instead of clicking on each game.

Thanks for reading have a nice day.

[Issue created by [deleted]: 2019-08-08]
